README.md

```diff
@@ -44,6 +63,30 @@ There is a huge space for subjective and personal preferences here, and it would
 
 A limitation originates from the treatment of comments by the OCaml parser: they are not included in the parse tree itself, but are only available as a separate list. This means that OCamlFormat must decide where to place the comments within the parse tree. It does this based on source locations of code and comments, as well as using a few heuristics such as whether only white space separates a comment and some code.
 
+### Options
+
+The full options' documentation is available in [ocamlformat-help.txt] and through `ocamlformat --help`.
+Options can be modified by the means of:
+- an .ocamlformat configuration file with an `option = VAL` line
+- using the `OCAMLFORMAT` environment variable: `OCAMLFORMAT=option=VAL,...,option=VAL`
+- an optional parameter on the command line
+- a global `[@@@ocamlformat "option=VAL"]` attribute in the processed file
+- an `[@@ocamlformat "option=VAL"]` attribute on expression in the processed file
+
+.ocamlformat files in current and all ancestor directories for each input file are used, as well as the global .ocamlformat file defined in `$XDG_CONFIG_HOME/ocamlformat`. The global .ocamlformat file has the lowest priority, then the closer the directory is to the processed file, the higher the priority.
```

Comment:
s/in current/in the containing/

I worry that users will misunderstand `current` to mean the current directory of the ocamlformat invocation, rather than the directory containing the input file.